src/app/modules/template/template-solution-list/template-solution-list.component.html

```diff
@@ -1,4 +1,9 @@
 <div class="mat-elevation-z8">
+
+  <button mat-button color="primary" (click)="goBack()" class="back-button">
```

Comment:
why are we giving color="primary" this could go to the primary of the angular material and maybe the reason why you had to use !important, remove this and try